test/conversion.spec.ts

```diff
@@ -0,0 +1,98 @@
+import 'mocha';
+import { expect } from 'chai';
+import { MassaAmount, MASSA_UNIT } from '../src';
+
+describe('MassaAmount Class', () => {
+  it('classical unit test conversion cases', () => {
+    const amount = new MassaAmount(1, MASSA_UNIT.MASSA);
```

Comment:
I would have created a Massa(), MilliMassa() and NanoMassa() and add basic operators to MassaAmount to offer a way to write cleaner code such as:

```typescript
const amount = 1 * Massa()
```
Furthermore, calculus on amount would have been possible.